numpy/core/tests/test_indexing.py

```diff
@@ -981,10 +981,12 @@ def _get_multi_index(self, arr, indices):
                         try:
                             mi = np.ravel_multi_index(indx[1:], orig_slice,
                                                       mode='raise')
-                        except Exception:
+                        except Exception as e:
```

Comment:
I think @eric-wieser 's comment about not modifying the tests was intended to apply to all tests, not only those in `test_regression.py`.